packages/cashier/src/pages/account-transfer/account-transfer-receipt/account-transfer-receipt.tsx

```diff
@@ -99,11 +99,9 @@ const AccountTransferReceipt = observer(({ onClose, history }: TAccountTransferR
                             <Localize i18n_default_text={selected_from.text} />
                         </Text>
                     </div>
-                    {!(is_from_derivgo && selected_from.is_derivez) && (
-                        <Text as='p' size='s' color='less-prominent' align='center'>
-                            {selected_from.value}
-                        </Text>
-                    )}
+                    <Text as='p' size='s' color='less-prominent' align='center'>
```

Comment:
`non-blocking comment:` remove the default size s here 👍  
```suggestion
                    <Text as='p' color='less-prominent' align='center'>
```